bip-0077.md

```diff
@@ -734,3 +738,15 @@ directory, and development kit may be found here:
 <https://github.com/payjoin/rust-payjoin>. Source code for an Oblivious
 HTTP relay implementation may be found here:
 <https://github.com/payjoin/ohttp-relay>.
+
+## Changelog
+
+- 0.2.0 2025-06-08
```

Comment:
I was looking at the diff from the force push and noticed that it was July before and now is June. Just wanted to doublecheck whether you meant June.
```diff
-- 0.2.0 Jul 8 2025
+- 0.2.0 2025-06-08
```